demo/project.godot

```diff
@@ -12,7 +12,7 @@ config_version=5
 
 config/name="OpenXR Demo"
 run/main_scene="res://main.tscn"
-config/features=PackedStringArray("4.2", "GL Compatibility")
+config/features=PackedStringArray("4.3", "GL Compatibility")
```

Comment:
Can you also bump the minimum version in `plugin.gdextension`

Reply:
Ah yes, nice catch. Done!